### PR TITLE
chore(setup-redux): Setting up redux for the project

### PR DESCRIPTION
**Pull request by myrdstom**
#### What does this PR do?
This PR sets up the redux for the application.

#### Description of Task to be completed?
Setting up redux to manage the state 

- Install redux

- Create the folder structure to handle the store, reducers and actions

- Setup the redux store

#### How should this be manually setup?

- Run the command below;
  `yarn install`

#### Relevant pivotal tracker stories

#162685617

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,8 +1,5 @@
 import React, { Component } from 'react';
 
-
-const App = () => (
-    <div>Hello World</div>
-);
+const App = () => <div>Hello World</div>;
```
Comment (fahadmak) [inline src/components/App.js]: Wouldn't it be more readable with parenthesis

File: src/store/combineStore.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+
+export default () => {
+  const store = createStore(combineReducers({}), applyMiddleware(thunk));
```
Comment (fahadmak) [inline src/store/combineStore.js]: create a function to return a combined reducer because you may be combining more than ten reducers

**Review by fahadmak — CHANGES_REQUESTED** [review]

**Comment by fahadmak** [comment]
LGTM

**Review by VivianDoreen — APPROVED** [review]
LGTM with the exception of circleCI

**Review by VivianDoreen — COMMENTED** [review]
CircleCI is still failing, I think this will be successful after @moseskamira  is merged and @myrdstom rebases.

**Review by fahadmak — APPROVED** [review]

**Pull request merged**